toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -351,72 +352,72 @@ func customizeOSContents(ic *ImageCustomizerParameters) error {
 	// images at this time because such modifications would compromise the integrity and security mechanisms enforced by dm-verity.
 	err := checkDmVerityEnabled(ic.rawImageFile)
 	if err != nil {
-		return err
+		return [UuidSize]byte{}, "", err
 	}
 
 	// Customize the partitions.
 	partitionsCustomized, newRawImageFile, partIdToPartUuid, err := customizePartitions(ic.buildDirAbs,
 		ic.configPath, ic.config, ic.rawImageFile)
 	if err != nil {
-		return err
+		return [UuidSize]byte{}, "", err
 	}
 	ic.rawImageFile = newRawImageFile
 
 	// Create a uuid for the image
 	imageUuid, imageUuidStr, err := createUuid()
```

Comment:
I think the code might be a little simpler if the call to `createUuid` was moved into `createImageCustomizerParameters`, adding `imageUuid` and `imageUuidStr` to `ImageCustomizerParameters`.